app/views/parks/index.html.erb

```diff
@@ -1,15 +1,13 @@
+<h1> Dog Parks </h1>
+<h3 class="hollow button"><%= link_to "Add a New Park", new_park_path %> </h3>
```

Comment:
Please fix the indentation in this file -- unsure if there are actually any meaningful changes or whether it's just all whitespace changes